docs/blog/things-to-know-when-developing-with-lit-element/index.md

```diff
@@ -0,0 +1,233 @@
+
+You aren't able to use numbers directly, no. However, the foot gun of lacking clarity in your code that unearths makes preventing this input a pretty good idea. In a world where `px` is just one of many units that a CSS property could accept (`%`, `vh`, `vwmax`, `pt`, `em`, `rem`, `pt`, `pc`, ad (almost) infinitum) being explicit with your code with not only help your consumers, but it'll help your future self maintain the components that you create. On top of that, there are numbers that you'll eventually want to apply to your styles that won't beg for the application of a unit at all:
+
+<figure>
+
+```js
+class XL extends LitElement {
+  static styles = css`
+    output {
+      background-color: hsl(var(--hue, 0) 50 100);
+    }
+  `;
+  @property({ type: Number }) hue = 0;
+  render() {
+    return html` <output style=${styleMap({ '--hue': this.hue })}></output> `;
+  }
+}
+```
+
+<figcaption>
+Thanks @bennypowers for the code sample here.
+</figcaption>
+</figure>
+
+---
+
+One of the best things about `lit-html` the renderer underlying LitElement is that, if you want to live on that wild side, you could easily create your own directive that applied number typed properties as `px` and leverage it in your own work. Here are [the docs](https://lit.dev/docs/templates/custom-directives/) for doing just that! If you're still not convinced, check out some [directives with which I've experimented](https://dev.to/open-wc/doing-a-flip-with-lit-html-2-0-3gn4).
+
+NOTE: the above article, "Doing a FLIP with lit-html@2.0", was written against an RC of `lit@2.0` and may not be 100% current. I'll be looking to update it here, soon.
+
+## Conclusion
+
+When you're learning a new piece of software, by all means, start by comparing it to something that you know. It's like a cheat code to getting started down the path of learning something new. Once you've done that, don't stop there, get into a real use case with it and learn what sort of capabilities or techniques it unlocks or supports. Only then can you really get into the question of why it's doing so and whether in the context that it should be used (or the context that you might use it) it's the sort of tool you want to leverage for the job.
```

Comment:
"whether in the context that it should be used " -> "whether in the context that it is intended to be used "